src/windows/mod.rs

```diff
@@ -497,7 +497,7 @@ extern "C" {
     pub fn aligned_free(ptr: *mut ::c_void);
 }
 
-extern "system" {
+extern "stdcall" {
```

Comment:
`system` is already equivalent to `stdcall` on 32-bit x86. On all other platforms it is equivalent to `C`. All this change does it break non-x86 platforms.